simtools/job_submission/job_manager.py

```diff
@@ -0,0 +1,207 @@
+import logging
+import os
+from copy import copy
+
+import simtools.util.general as gen
+
+__all__ = ["JobManager"]
+
+class MissingWorkloadManager(Exception):
+    pass
+
+class JobManager:
+    """
+    JobManager provides an interface to workload managers
+    like gridengine or HTCondor.
+
+    Attributes
+    ----------
+    submitCommand: str
+       job submission command (allowed is qsub, condor_submit, seriell_script
```

Comment:
The ``more`` option in the code is not described here (also I am not entirely sure what it achieves, see below).

Reply:
We can actually drop the `more` option, as we have the `--test` option (from the command line). This does not submit anything, but prints all the file names to the screen. One can easily check than that e.g., the sim_telarray command is correct and that the model has been exported correctly. 

I've remove everything related to `more`